scio-bigtable/src/main/java/com/spotify/scio/bigtable/BigtableUtil.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2016 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.bigtable;
+
+import com.google.bigtable.repackaged.com.google.cloud.config.BigtableOptions;
+import com.google.bigtable.repackaged.com.google.cloud.grpc.io.ChannelPool;
+import com.google.bigtable.repackaged.com.google.com.google.bigtable.admin.v2.Cluster;
+import com.google.bigtable.repackaged.com.google.com.google.bigtable.admin.v2.ListClustersRequest;
+import com.google.bigtable.repackaged.com.google.com.google.bigtable.admin.v2.BigtableInstanceAdminGrpc;
+import com.google.bigtable.repackaged.com.google.com.google.bigtable.admin.v2.BigtableInstanceAdminGrpc.BigtableInstanceAdminBlockingStub;
+import com.google.bigtable.repackaged.com.google.com.google.bigtable.admin.v2.ListClustersResponse;
+import com.google.cloud.bigtable.dataflow.CloudBigtableConfiguration;
+import org.joda.time.Duration;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Utilities to deal with Bigtable.
+ */
+public final class BigtableUtil {
+
+  public static final Duration DEFAULT_SLEEP_DURATION = Duration.standardMinutes(20);
+
+  private BigtableUtil() { }
+
+  /**
+   * Updates all clusters within the specified Bigtable Instance to have a specified number
+   * of nodes. Useful for increasing the number of nodes at the start of the job and decreasing
+   * it at the end to lower costs yet still get high throughput during bulk ingests/dumps.
+   *
+   * @param cloudBigtableConfiguration Configuration of Bigtable Instance
+   * @param numberOfNodes Number of nodes to make cluster
+   * @param sleepDuration How long to sleep after updating the number of nodes. Google recommends
+   *                      at least 20 minutes before the new nodes are fully functional
+   * @throws IOException If setting up channel pool fails
+   * @throws InterruptedException If sleep fails
+   */
+  public static void updateNumberOfBigtableNodes(final CloudBigtableConfiguration cloudBigtableConfiguration,
```

Comment:
This now has a wrapper in the BigtableScioContext which allows this to be ignored in tests. However this method is still public so it is confusing to know which one to use. Package private doesn't work here. Any suggestions @nevillelyh ?